.circleci/config.yml

```diff
@@ -173,6 +184,16 @@ workflows:
                       branches:
                           only: develop
 
+            - run-ui-tests:
+                requires:
+                    - deploy-site-to-test
+                context: tfn-fdbt-automation-test
+                filters:
+                      branches:
+                          only:  
+                              - develop
+                              - master
```

Comment:
I'd remove master, we only really need the tests after deploying to test and it won't work since the deploy-site-to-test job won't exist on the master pipeline

Reply:
Resolved